packages/ast-spec/src/declaration/VariableDeclaration/spec.ts

```diff
@@ -11,7 +11,8 @@ export interface LetOrConstOrVarDeclaration extends BaseNode {
   /**
    * The variables declared by this declaration.
    * Note that there may be 0 declarations (i.e. `const;`).
-   * ```
+   * @example
+   * ```ts
```

Comment:
I tried to make sure all <code>```</code>s had a language and were in a proper `@example`, so they'd play nicely with docs generation. But multiline ones expose a bug in the markdown table output 🫤:

![Screenshot of the LetOrConstVariableDeclaration where only the first line of the example code blocks are actual markdown codeblocks](https://github.com/typescript-eslint/typescript-eslint/assets/3335181/87f8ad3d-2b37-438a-8a1c-b07c75e58d1e)

Reply:
Resolved by splitting into individual `@example`s. 🤷

Reply:
Some of them are still a little messy... but I don't know the right way to get around this. 😬 putting in review.